README.md

```diff
@@ -1,127 +1,224 @@
-# Table of Contents
+# Name
 
-- [Table of Contents](#table-of-contents)
-  - [Name](#name)
-  - [Synopsis](#synopsis)
-  - [Methods](#methods)
-    - [new](#new)
-    - [Path](#path)
-      - [Full path match](#full-path-match)
-      - [Prefix match](#prefix-match)
-      - [Parameters in path](#parameters-in-path)
-    - [match](#match)
-    - [dispatch](#dispatch)
-  - [Install](#install)
-    - [Compile and install](#compile-and-install)
-  - [DEV ENV](#dev-env)
-    - [Install Dependencies](#install-dependencies)
-  - [Benchmark](#benchmark)
-
-## Name
-
-This is Lua implementation library base on FFI for [rax](https://github.com/antirez/rax).
+Radix tree implementation based on [rax](https://github.com/antirez/rax).
+
+## Status
 
 [![Build Status](https://github.com/api7/lua-resty-radixtree/actions/workflows/test.yml/badge.svg)](https://github.com/api7/lua-resty-radixtree/actions/workflows/test.yml)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/iresty/lua-resty-radixtree/blob/master/LICENSE)
 
-This project depends on [lua-resty-ipmatcher](https://github.com/api7/lua-resty-ipmatcher) and [lua-resty-expr](https://github.com/api7/lua-resty-expr).
+Dependencies:
+
+- [lua-resty-ipmatcher](https://github.com/api7/lua-resty-ipmatcher)
+- [lua-resty-expr](https://github.com/api7/lua-resty-expr)
 
-This project has been working in microservices API gateway [Apache APISIX](https://github.com/apache/incubator-apisix).
+Used by:
 
-The project is open sourced by [API7.ai](https://api7.ai/) Inc.
+- [Apache APISIX](https://github.com/apache/apisix): A high-performance cloud native API gateway.
 
-In addition to this open source version, our company also provides a more powerful and performing commercial version, and provides technical support. If you are interested in our commercial version, please [contact us](https://www.apiseven.com/).
+Developed by [API7.ai](https://api7.ai/).
 
-## Synopsis
+> **Note**
+>
+> API7.ai provides technical support for the software it maintains like this library and [Apache APISIX](https://github.com/apache/apisix). Please [contact us](https://api7.ai/contact) to learn more.
+
+# Table of Contents
```

Comment:
GitHub README automatically renders a ToC :) I prefer remove this part.

Reply:
But it does not make it easy to navigate for documentation. I think it is easier to navigate from a TOC in the readme. Also having "back to TOC" links makes it easy to switch between docs of different methods and examples.